dev_guide/builds.adoc

```diff
@@ -659,6 +659,30 @@ code. If your application exists inside a sub-directory, you can override the
 default location (the root folder) using this field.
 ====
 
+[[max-duration]]
+
+== Build maximum duration
+When defining a `*BuildConfig*`, you can define its maximum duration, setting
+`*maxDuration*` field, it is specified in seconds and by default it is not set.
```

Comment:
"When defining a BuildConfig, you can define its maximum duration by setting the maxDuration field. It is specified in seconds and is not set by default."